net/ddns-scripts/files/usr/lib/ddns/dynamic_dns_functions.sh

```diff
@@ -707,6 +707,9 @@ do_transfer() {
 
 	[ $# -ne 1 ] && write_log 12 "Error in 'do_transfer()' - wrong number of parameters"
 
+	# Use ip_network as default for bind_network if not separately specified
+	[ -z "$bind_network" -a "$ip_source" = "network" -a "$ip_network" ] && bind_network="$ip_network"
```

Comment:
I am aware that in the whole script -a or -o is used.
But can you please change that to && this is more POSIX comform.
`[ -z "$bind_network" ] && [ "$ip_source" = "network" ] && [ "$ip_network" ] && bind_network="$ip_network"`
But I think this is wrong----------------------------------------------------------------^  ?

I also do not verify if this is correct, because I do not use this.
@Ansuel Can you say something about this

Reply:
A better question would be...
@mrjoel what does this fix? Why this change?

Reply:
Found the issue... It would be good to add to the commit description
"Fixes: #13182"

Reply:
@feckert looks like this fix a specific problem...
bind_network is indeed a separate ddns config but is needed when ip_network is used...

So i think we can accept this when the `if` is fixed to follow POSIX and a "Fixed" tag is added to the commit description (Commit description is different than the pr description...)

Reply:
Updated for better POSIX idioms and added issue link to commit message.